app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/NightscoutUploader.java

```diff
@@ -55,24 +59,43 @@
         private Boolean enableRESTUpload;
         private Boolean enableMongoUpload;
         private SharedPreferences prefs;
+        private OkHttpClient client;
+
+        public interface NightscoutService {
+            @POST("entries")
+            Call<ResponseBody> upload(@Header("api-secret") String secret, @Body RequestBody body);
+
+            @POST("entries")
+            Call<ResponseBody> upload(@Body RequestBody body);
+
+            @POST("devicestatus")
+            Call<ResponseBody> uploadDeviceStatus(@Body RequestBody body);
+
+            @POST("devicestatus")
+            Call<ResponseBody> uploadDeviceStatus(@Header("api-secret") String secret, @Body RequestBody body);
+
+        }
+
+        private class UploaderException extends RuntimeException {
+            int code;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1306138886/issues/source?bid=2735053&fileBranchId=2947762#l80)